apps/src/templates/CensusForm.jsx

```diff
@@ -41,15 +42,36 @@ const styles = {
 class CensusForm extends Component {
 
   state = {
-    value: ''
+    submission: {
+      name: '',
+      email: '',
+      role: '',
+      pledge: false,
+    }
   };
 
-  handleChange = event => {
-    this.setState({value: event.target.value});
+  handleChange = (propertyName, event) => {
+    const submission = this.state.submission;
+    submission[propertyName] = event.target.value;
+    this.setState({submission: submission});
   }
 
-  handleSubmit(event) {
-    alert('You clicked submit!');
+  processResponse() {
+    console.log("submission success!");
+  }
+
+  processError() {
+    console.log("error submittiing");
+  }
```

Comment:
better to print out the error object received by this function, e.g.
```
processError(error) {
  console.log("error submitting", error);
} 
```
or possibly `console.log(JSON.stringify(error, null, 2))` if that's easier to read. This will help you figure out what validation rules to put on each line that you write in the Form class, since you'll be able to see what the problems are when they get rejected.